query/lambdas/load_data/load_data.py

```diff
@@ -0,0 +1,35 @@
+import json
+
+from query.lib.etl.transform import BundleDocumentTransform
+from query.lib.config import Config
+from query.lib.db.database import PostgresDatabase
+from query.lib.etl.extract import S3Extractor, Extractor
+from query.lib.etl.load import PostgresLoader, Loader
+from query.lib.etl.s3_client import S3Client
+from query.lib.common.logging import get_logger
+
+logger = get_logger('query.lambdas.load_data.load_data')
+
+# TODO: don't hardcode to staging, don't read directly from s3
+extractor = S3Extractor(S3Client('us-east-1', 'org-hca-dss-staging'))
+loader = PostgresLoader(PostgresDatabase(Config.serve_database_uri))
+
+
+class LoadData:
+    def query_service_data_load(self, event, context):
```

Comment:
You can better separate the concerns of `query_service_data_load` from lambda by

1. Moving the unpacking of `event` to the `load_data_lambda.app.load_data` method
1. Removing the `context` parameter. This is not used.

Reply:
👍